### DISCUSSION TITLE
A question about IEA-15-240-RWT-UMaineSemi in OpenFAST

### DISCUSSION THREAD

**Discussion by LeoHongLiZuo** [OpenFAST/openfast#2020] [Q&A]
Dear guys
I have a question about IEA-15-240-RWT-UMaineSemi in OpenFAST.
As shown in the following figure, I want to add appropriate AXIAL COEFFICIENTS (such as AxCd) to the floating platform in HydroDyn.
<img width="416" alt="111" src="https://github.com/OpenFAST/openfast/assets/108517989/c834affc-12fb-4bdd-a2fe-fb9ec6003723">
But I have not found the appropriate value, does anyone know how to define the parameter there.
Thanks a lot!

**Comment by jjonkman** [comment]
Dear @LeoHongLiZuo,

If you are asking about how `AxCd` in HydroDyn is defined, please see the HydroDyn documentation on readthedocs: https://openfast.readthedocs.io/en/main/source/user/hydrodyn/input_files.html#axial-coefficients and https://openfast.readthedocs.io/en/main/_downloads/39aafbb61663ae16bd1a7e80f305c20b/HydroDyn_Plan_TCF_Morison.docx.

If you are asking what value specifically should be used for the VolturnUS semisubmersisble supporting the IEA Wind 15-MW reference wind turbine, the public version of this model (available here: https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-UMaineSemi) is not released with strip-theory members; rather, the model is released with an equivalent quadratic drag matrix to approximate viscous effects.  A strip-theory implementation of viscous drag is preferred over a quadratic drag matrix, but I'm not aware of the specific drag coefficients appropriate for this model.  That would probably be a better question to ask within the IEA Wind 15-MW reference wind turbine repository.

Best regards,
